JustFakeIt/ExpectationExtensions.cs

```diff
@@ -53,7 +56,48 @@ public static bool MatchesActualHttpMethod(this HttpRequestExpectation expected,
 
         public static bool MatchesActualBody(this HttpRequestExpectation expected, string actualBody)
         {
-            return string.IsNullOrEmpty(expected.Body) || actualBody.Equals(expected.Body);
+            return MatchBody(expected.Body, actualBody);
+        }
+
+        public static bool MatchBody(string expectedBody, string actualBody)
+        {
+            return string.IsNullOrEmpty(expectedBody) || actualBody.Equals(expectedBody) || expectedBody.JsonKeyValuesMatch(actualBody);
+        }
+
+        public static bool JsonKeyValuesMatch(this string expectedBody, string actualBody)
+        {
+            JObject expectedJObject;
+            JObject actualJObject;
+
+            try
+            {
+                expectedJObject = JsonConvert.DeserializeObject<JObject>(expectedBody);
+                actualJObject = JsonConvert.DeserializeObject<JObject>(actualBody);
+            }
+            catch (Exception)
+            {
+                return false;
```

Comment:
That's to try the parsing method. If the string is not a valid json, it throws, so we can stop evaluating and we know we don't need to match json.